theano/gpuarray/opt.py

```diff
@@ -1298,7 +1291,7 @@ def local_inplace_sparseblockouter(node):
 
 
 # This deals with any abstract convs that have a transfer somewhere
-@register_opt('fast_compile', 'conv_dnn')
```

Comment:
why remove this? I think it should be kept and we should have the cudnn tag too.